src/go/intelligence/pipeline/keyword_matching.go

```diff
@@ -0,0 +1,90 @@
+package pipeline
+
+import (
+	"go/intelligence/models"
+	"go/intelligence/utils"
+	"strings"
+)
+
+// Module     : intelligence
+// Author     : Rahul Indra <indrarahul2013 AT gmail dot com>
+// Created    : Wed, 1 July 2020 11:04:01 GMT
+// Description: CERN MONIT infrastructure Intelligence Module
+
+//KeywordMatching function finds defined keywords in the shortDescription of alerts and assign severity level accordingly
+func KeywordMatching(data <-chan models.AmJSON) <-chan models.AmJSON {
+
+	dataWithSeverity := make(chan models.AmJSON)
+	go func() {
+		for each := range data {
+			if each.Labels["service"] == "SSB" {
```

Comment:
This should be generalized via local map and reflection.

Reply:
Added this.